### ISSUE TITLE
Steam Crashes on Startup - Segfault

### ISSUE THREAD

**Issue by bryonadams** [ValveSoftware/steam-for-linux#4845]
#### Your system information

* Steam client version (build number or date): 1.0.0.54-3.1  
* Distribution (e.g. Ubuntu): openSUSE Leap 42.2 (Gnome)
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
I launch steam and it crashes. Running steam --reset (to reset it I guess? The man page is basically empty) doesn't change anything. Output in link below:
https://gist.github.com/bryonadams/081a1c92a843141825c660ebcf0faedc

Edit: I did a clean uninstall and the issue persists. ````rpm -ql steam```` to find directories (and confirm they're deleted) and deleted ````.local/share/Steam```` before reinstalling from packman repo.

Edit 2: This keeps showing up in the log when I launch Steam: 
````Feb 04 21:21:48 coffee-to-go.bryonadams.com kernel: steam[19177]: segfault at f52aa193 ip 00000000f62c1ce5 sp 00000000ff8667cc error 7 in libcrypto.so.1.0.0[f627b000+1c3000]````
Issue with libcrypto that Steam doesn't like?

#### Steps for reproducing this issue:

1. Launch Steam
2. 
3.

**Comment by bryonadams** [comment]
If there's additional information I can provide please let me know. There's a dmp file in the log I linked to, though I don't know how to read it.

**Comment by kisak-valve** [comment]
Hello @bryonadams,
What video driver vendor/version are you using?
If you run `STEAM_RUNTIME_PREFER_HOST_LIBRARIES=0 steam` from a terminal, does it change the behavior?

I currently do not have openSUSE on a test box right now, but there was some steam runtime compatibility trouble a little while back with using mesa built against openssl which was worked-around by using mesa built against libnettle (#4504) , but the openSUSE package maintainer(s) for steam may have opted to go with a different solution since then. Additionally, you may be affected by https://bugzilla.opensuse.org/show_bug.cgi?id=1021095.

**Comment by bryonadams** [comment]
Hi @kisak-valve , 
I'm using the integrated graphics on a i5-5300U. It appears I'm using the i915 kernel driver.

Output from lscpi -v:
```
00:02.0 VGA compatible controller: Intel Corporation Broadwell-U Integrated Graphics (rev 09) (prog-if 00 [VGA controller])
	Subsystem: Dell Device 062e
	Flags: bus master, fast devsel, latency 0, IRQ 46
	Memory at f6000000 (64-bit, non-prefetchable) [size=16M]
	Memory at e0000000 (64-bit, prefetchable) [size=256M]
	I/O ports at f000 [size=64]
	Expansion ROM at <unassigned> [disabled]
	Capabilities: <access denied>
	Kernel driver in use: i915
	Kernel modules: i915
```
If I set ````STEAM_RUNTIME_PREFER_HOST_LIBRARIES=0```` from a terminal and then launch Steam I have the same error it looks like.

I'll take a look at the other issue and bugzilla you referenced.

**Comment by bryonadams** [comment]
@kisak-valve I was able to reproduce this reliably, though I only have the one laptop to test on. I edited my $PATH in my .bashrc using: ````export PATH="/home/bryon/bin:/usr/local/bin:/usr/bin:/bin:/usr/bin/X11:/usr/games:/sbin:/usr/sbin"````

openSUSE by default doesn't include ````/usr/sbin```` or ````/sbin````, I added them to make it easier to run a few things that don't actually need root such as traceroute or reboot.

Perhaps there is a variable somewhere in the Steam bash file that is messed up by those paths?
